charts/flux2/templates/helm-controller.crds.yaml

```diff
@@ -4,7 +4,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.12.0
-  creationTimestamp: null
```

Comment:
this removal  is a result of running `make generate` , not manual.  Could be due to the kustomize version on my local workstation, which is:
```
$ kustomize version
{Version:kustomize/v4.5.2 GitCommit:9091919699baf1c5a5bf71b32ca73a993e98088b BuildDate:2022-02-09T23:19:28Z GoOs:darwin GoArch:arm64}
```
anyways, `creationTimestamp: null` was also removed from upstream : https://github.com/kubernetes-sigs/controller-tools/pull/800 . so, this is good riddance here also..